api/notd/store/schema.py

```diff
@@ -139,6 +139,7 @@
     sqlalchemy.Column(key='address', name='address', type_=sqlalchemy.Text, nullable=False),
     sqlalchemy.Column(key='date', name='date', type_=sqlalchemy.DateTime, nullable=False),
     sqlalchemy.Column(key='transferCount', name='transfer_count', type_=sqlalchemy.Numeric(precision=256, scale=0), nullable=False),
+    sqlalchemy.Column(key='mintCount', name='mint_count', type_=sqlalchemy.Numeric(precision=256, scale=0), nullable=False),
```

Comment:
you need to update the actual schema

Reply:
this all needs to be optional for now cos its gonna take AGES to back fill this. Make another PR (wip) that sets it back to not optional so I can merge that once the backfilling is done.

Reply:
Okay